resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/DriverConfigurationStepsOrchestrator.scala

```diff
@@ -94,6 +99,22 @@ private[spark] class DriverConfigurationStepsOrchestrator(
         submissionSparkConf)
     val kubernetesCredentialsStep = new DriverKubernetesCredentialsStep(
         submissionSparkConf, kubernetesResourceNamePrefix)
+    val hadoopConfigurations = hadoopConfDir.map(conf => getHadoopConfFiles(conf))
```

Comment:
Can we parse out the hadoop configuration files in the step and not in the orchestrator?